core/templates/dev/head/pages/topic_editor/TopicEditorNavbarDirective.js

```diff
@@ -23,20 +23,25 @@ oppia.directive('topicEditorNavbar', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/topic_editor/topic_editor_navbar_directive.html'),
       controller: [
-        '$scope', '$uibModal', 'AlertsService',
+        '$scope', '$rootScope', '$uibModal', 'AlertsService',
         'UndoRedoService', 'TopicEditorStateService',
-        'TopicRightsBackendApiService',
+        'TopicRightsBackendApiService', 'TopicEditorRoutingService',
         'EVENT_TOPIC_INITIALIZED', 'EVENT_TOPIC_REINITIALIZED',
         'EVENT_UNDO_REDO_SERVICE_CHANGE_APPLIED',
         function(
-            $scope, $uibModal, AlertsService, UndoRedoService,
+            $scope, $rootScope, $uibModal, AlertsService, UndoRedoService,
             TopicEditorStateService, TopicRightsBackendApiService,
+            TopicEditorRoutingService,
```

Comment:
Line breaks here should match up with the line breaks in the stringified dependencies.

/cc @apb7 @kevinlee12 -- this is important to have a lint check for. We've had failures before due to mismatches.

Reply:
Done.